src/ert/_c_wrappers/job_queue/__init__.py

```diff
@@ -59,6 +59,7 @@ def setenv(var: str, value: str) -> None:
     setenv("LSF_SERVERDIR", f"{LSF_HOME}/etc")
     setenv("LSF_ENVDIR", f"{LSF_HOME}/conf")  # This is wrong: Equinor: /prog/LSF/conf
 
+# pylint: disable=wrong-import-position
```

Comment:
Is this for circumventing a circular import, or do the imports depend on environment variables? Maybe a comment is warranted.

Reply:
This is probably needed in the scenario where ERT is run on a Linux node with direct access to the LSF libraries and not through $ERT_LSF_SERVER. This probably amounts to zero user accessible nodes in Equinor, but pruning that functionality from the driver and thus these lines, belongs to a different PR. Keep for now.